### ISSUE TITLE
Not able to use bottom nav component with router-link

### ISSUE THREAD

**Issue by IniZio** [museui/muse-ui#28]
#### I tried to use bottom nav item as router-link, but numerous methods did work.
They include:
- `<router-link :to="'home'"><mu-bottom-nav-item/></router-link>`:  seems to break the css properties of nav item, making them squeeze together instead of even width
- `<mu-bottom-nav-item title="Recents" icon="restore"><router-link :to="'home'" /></mu-bottom-nav-item>`: css properties seems normal, but clicking on it does not trigger router-link
- `<router-link :to="'home'" tag="mu-button-nav-item" icon="restore"  />`: does not appear at all, and props cannot be passed into nav item, so cannot show the font icon.

Does Muse UI support router-link or what is the recommended way to use them together? I am currently using workaround of watching the value of mu-bottom-nav instead.

**Comment by BubblyPoker** [comment]
you can use `this.$router.push()` instead at present

**Comment by myronliu347** [comment]
`mu-bottom-nav-item` will add  href property  in next version, you can use it

```html
<mu-button-nav-item href="#/home" icon="restore" title="restore" />
```

now，please use `this.$router.push()` on change

**Comment by IniZio** [comment]
Thanks for the clarification, can close now or wait until the feature is added in next version

**Comment by myronliu347** [comment]
fix in 2.0.0-alpha.5, please update

**Comment by IniZio** [comment]
Thanks :+1:
